### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by ganemone**

### CODE REVIEW

**Comment by ganemone** [comment]
+ @divyenduz

File: package.json [hunk 1]
```diff
@@ -148,7 +148,7 @@
     "graphql": "14.4.2",
     "graphql-config": "2.2.1",
     "graphql-config-extension-prisma": "0.3.0",
-    "graphql-language-service-server": "2.1.0",
+    "graphql-language-service-server": "^2.3.0",
```
Comment (divyenduz) [inline package.json]:
```suggestion
    "graphql-language-service-server": "2.3.0",
```

**Comment by ganemone** [comment]
Ping @divyenduz

**Comment by ScreamZ** [comment]
@divyenduz Ping 🙏🏻

**Comment by divyenduz** [comment]
Sorry about the delay 🙏
Accepting PRs for renovate to bump deps and automerge them 🙏

**Pull request merged**